### PR TITLE
Volnag python

### PR DESCRIPTION
**Pull request by volnagPython**
Trying to push my local main branch to the remote repository main branch on GitHub

### CODE REVIEW

File: HW2/Zen.txt
Comment (lhalam) [inline HW2/Zen.txt:1]: перемісти у відповідну директорію `hw/hw02/volnagPython`

File: HW2/screenShot.jpg
[the remark this answers is not in the document]
Comment (volnagPython) [reply HW2/screenShot.jpg:1]: Що перемістити?

**Comment by volnagPython** [comment]
I made the required changes in the local repository and try to push "local main" branch  to https://github.com/UA-1030-PythonFundamentals/PythonFundamentals.git main again. It was rejected . It looks like a problem with Readme.md file.

**Comment by volnagPython** [comment]
Started a new pull request to push my local main branch to the remote repository main branch.  My local branch volnagPython and my remote branch volnagPython on GitHub are already synchronized. Trying to do the same with main branches.

**Review by lhalam — APPROVED** [review]

**Pull request merged**